lib/src/widgets/chat/ds_location_message_bubble.widget.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class DSLocationMessageBubble extends StatelessWidget {
+  final DSAlign align;
+  final DSMessageBubbleStyle style;
+  final String? title;
+  final String latitude;
+  final String longitude;
+
+  DSLocationMessageBubble({
+    super.key,
+    required this.align,
+    required this.latitude,
+    required this.longitude,
+    DSMessageBubbleStyle? style,
+    this.title,
+  }) : style = style ?? DSMessageBubbleStyle();
+
+  final appKey = 'AIzaSyAlC3a3DZZBscR0QIbQpee13Op9Y05m_wc';
+
+  @override
+  Widget build(BuildContext context) {
+    final foregroundColor = style.isLightBubbleBackground(align)
+        ? DSColors.neutralDarkCity
+        : DSColors.neutralLightSnow;
+
+    return GestureDetector(
+      onTap: () => launchUrl(
+        Uri.parse(
+          'https://www.google.com/maps/search/?api=1&query=$latitude,$longitude',
+        ),
+      ),
+      child: DSMessageBubble(
+        shouldUseDefaultSize: true,
+        defaultMaxSize: 400.0,
```

Comment:
Looking at the prototype, it looks like this bubble should have a fixed size of 240px. So, you should override both ```defaultMaxSize``` and ```defaultMinSize``` to 240.0.